### ISSUE TITLE
lang attribute missing from html element

### ISSUE THREAD

**Issue by keithgee** [DSpace/dspace-angular#339]
Setting the lang attribute for the html is an accessibility aid in that it allows certain screen readers for blind or visually impaired users to automatically adjust pronunciation and accents according to the value of this attribute.

There's a brief post about using this attribute here:
https://developer.paciellogroup.com/blog/2016/06/using-the-html-lang-attribute/

The attribute can be set on individual elements to override enclosing elements. 

I suggest setting the lang attribute for the html element on each page to match the i18n setting configured in DSpace. DSpace also allows language codes for individual metadata values, so on pages that display metadata, I also suggest setting the lang attribute for each metadata value with a language that does not match the default language for the site.

**Comment by lhenze** [comment]
Hi, I wonder if the second point @keithgee mentioned merits its own ticket?   
"setting the lang attribute for each metadata value with a language that does not match the default language for the site."
Or, rewording the title to include this aspect?

**Comment by tdonohue** [comment]
@lhenze : yes, it probably should be its own ticket.  Would you be willing to create a separate ticket for it?  Please feel free to create tickets in this GitHub project as you see fit.  We also can add custom labels if we'd like to group these tickets better (e.g. by `accessibility` or similar).  Feel free to ping me in Slack or GitHub if there's custom labels you feel could help here.

**Comment by tdonohue** [comment]
Closing, this appears to be fixed in 7.x now.  I see we have `<html lang="en">` by default, and it updates based on the selected language, e.g. selecting German updates it to be `<html lang="de">`.
